index.js

```diff
@@ -1,5 +1,14 @@
 'use strict';
 
 module.exports = {
-  name: 'ember-power-select-infinity'
+    name: 'ember-power-select-infinity',
+    contentFor: function(type, config) {
+        var emberPowerSelect = this.addons.filter(function(addon) {
+            return addon.name === 'ember-power-select';
+        })[0]
+        return emberPowerSelect.contentFor(type, config);
+    },
+    included: function(/* app */) {
```

Comment:
@bakerac4 is `included()` necessary here since we are just calling `_super()`.

Reply:
@billdami I believe so - https://github.com/aexmachina/ember-cli-sass#addon-usage